randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5197,12 +4824,6 @@ Extra - asset_id: collision_camera_081
   * Extra - actor_def: actordef:actors/props/totalrechargestation/charclasses/totalrechargestation.bmsad
   * Extra - start_point_actor_name: PRP_TotalR_Platform_000
   * Extra - start_point_actor_def: actordef:actors/props/weightactivatedplatform_total/charclasses/weightactivatedplatform_total.bmsad
-  > Tile Group (SCREWATTACK) 2
-      All of the following:
-          Before Artaria - Screw Attack Rotatable
-          Any of the following:
-              Space Jump or Simple IBJ
-              Walljump (Beginner) and Use Spin Boost
```

Comment:
This should instead connect to Under left side blocks with either Cross + Beginner IBJ, intermediate WBJ + Perform WBJ, Gravity, SJ + Morph/SA + Before Rotatable, or Simple IBJ + Before Rotatable.